### PR TITLE
Update links

### PR DESCRIPTION
**Pull request by MikolaB**

### CODE REVIEW

**Comment by sdiachen** [comment]
Move through those files and look for cross-references between those

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -24,11 +24,11 @@ As for now to get access to this chat please write email to [Sergii Diachenko](m
 
 ## How Can I Contribute? ##
 ### Reporting Bugs ###
-Log issue as __bug__ in the [issue tracker](https://bitbucket.org/VolleyManagement/volleymanagement/issues?status=new&status=open).
+Log issue as __bug__ in the [issue tracker](https://github.com/VolleyManagement/volley-management/issues?q=is%3Aopen+is%3Aissue).
 
 Before creating bug reports, please check existing open issues as you might find out that you don't need to create one. When you are creating a bug report, please include as many details as possible.
 ### Suggesting Improvements ###
-Log issue as __enhcancement__ or __proposal__ in the [issue tracker](https://bitbucket.org/VolleyManagement/volleymanagement/issues?status=new&status=open).
+Log issue as __enhcancement__ or __proposal__ in the [issue tracker](https://github.com/VolleyManagement/volley-management/issues?q=is%3Aopen+is%3Aissue).
```
Comment (sdiachen) [inline CONTRIBUTING.md]: Do we need a filter here?

Comment (MikolaB) [reply CONTRIBUTING.md]: I used just to be sure that we will get open and newest issues.
But it seems work such a way by default.
I can remove.

Comment (sdiachen) [inline CONTRIBUTING.md]: It glitching somehow :(

Comment (sdiachen) [reply CONTRIBUTING.md]: But yes - remove it please while we are at it

**Pull request merged**